korma/src/commonMain/kotlin/com/soywiz/korma/geom/vector/VectorBuilder.kt

```diff
@@ -128,6 +128,8 @@ fun VectorBuilder.arc(x: Double, y: Double, r: Double, start: Double, end: Doubl
 
 fun VectorBuilder.circle(x: Double, y: Double, radius: Double) = arc(x, y, radius, 0.0, PI.toDouble() * 2f)
 
+fun VectorBuilder.sphere(x: Double, y: Double, radius: Double) = arc(x, y, radius, 0.0, PI.toDouble() * 2f)
```

Comment:
I think this would be confusing when for example drawing a shape when using korge-3d. A sphere is 3 dimensional.
We have to figure out a way where inside the vector builder DSL don't have access to other stuff from upper levels.
I think https://github.com/Kotlin/kotlinx.html is doing this. Let's investigate